src/ConfigWrapper/WindowsRegistryConfigWrapper.cs

```diff
@@ -185,16 +222,26 @@ public void Set<T>(string key, T value, bool createKeyIfNeeded)
         /// <inheritdocs />
         public void Set<T>(string key, T value)
         {
-            using (var regKey = this.GetParentKey(key, false, true))
-            {
-                regKey.SetValue(this.GetKeyAsArray(key).Last(), value.ToString());
-                regKey.Close();
-            }
+            this.Set<T>(key, value, false);
         }
 
-        /// <inheritdocs/>
+        /// <inheritdocs />
         public void Delete(string key)
         {
+            this.Delete(key, false);
```

Comment:
Should the Delete method also consider the key as FullKey(key) as the Get method ?

Reply:
ok, I see the other method using isFullyQualified = false